example/android/build.gradle

```diff
@@ -1,11 +1,13 @@
 buildscript {
+    ext.kotlin_version = '1.3.50'
```

Comment:
Can update this to latest, not sure what it is but I know at least `1.3.70` has been released

Reply:
Okay! Thanks!

Reply:
Resolved in a later commit.